src/workspacefactory/wfactory_controller.js

```diff
@@ -102,6 +107,65 @@ WorkspaceFactoryController.MODE_JS = 'javascript';
 WorkspaceFactoryController.MODE_XML = 'xml';
 
 /**
+ * Creates a new toolbox for editing. Saves previously edited toolbox, and prompts
+ * user if previous toolbox has not been saved under a user-specified name. Initial
+ * default name of toolbox is the empty string, and is considered an un-named
+ * toolbox that the user will eventually rename if they add another toolbox.
+ */
+WorkspaceFactoryController.prototype.newToolbox = function() {
+  // TODO: implement
```

Comment:
Maybe consider a console.log(..) call to say the unimplemented method has been called.  Good for testing the UI is doing what you expect.

Here and below.